gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/GapicContext.java

```diff
@@ -18,13 +18,10 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.gapic.metadata.GapicMetadata;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
-import java.util.stream.Collectors;
+
 import javax.annotation.Nullable;
+import java.util.*;
```

Comment:
Wildcard importing is [not allowed](https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s3.3.1-wildcard-imports). You can change your Intellij setting to not automatically do it.